docs/cloud/provision/azure/considerations.md

```diff
@@ -1,31 +1,30 @@
 # Considerations for Microsoft Azure
 
-Due to differences between Microsoft Azure and other cloud providers, the provisioning process in Event Store Cloud is different from AWS and GCP. Event Store Cloud also has certain limitations when it comes to Azure, which you can also find on this page.
+Due to differences between Microsoft Azure and other cloud providers, the provisioning process in Event Store Cloud is different from AWS and GCP. We’ve made a list of these differences here in order to help you make an informed decision about Cloud providers.
 
 ## Network peering
 
-When creating peering links in Azure require the user to configure a security principle referencing the application ID of Event Store Cloud, as well as configure and apply a role allowing that principle to modify the network resource of the remote network. Event Store Cloud console provides the necessary Azure CLI commands at creation time, to support this operation.
+When creating peering links Azure require the user to configure a security principle referencing the application ID of Event Store Cloud, and also configure and apply a role allowing that principle to modify the network resource of the remote network. Event Store Cloud console provides the necessary Azure CLI commands at creation time, to support this operation.
 
 ## Disk resize
 
-Currently, it is impossible to expand disks on EventStoreDB cloud nodes provisioned to Azure. We plan to provide this feature at a later date, as an implementation for Azure requires a great deal more orchestration, as there is no native support within the platform itself.
+Currently, it is impossible to expand disks on EventStoreDB cloud nodes provisioned to Azure. We plan to provide this feature at a later date, but as there is no native support within Azure, implementing this requires more planning and thought on our part.
 
-We advise to choose the disk size accordingly from the start, so the volume size will accommodate the growth of your data over time. Another workaround would be to backup an existing cluster and restore the data to a new cluster with larger disks.
+There are positive ways around this, and we advise you to choose the disk size accordingly from the start, so the volume size will accommodate the growth of your data over time. You could also backup an existing cluster and restore the data to a new cluster with larger disks.
```

Comment:
Suggestions :
 
`You could also backup an exist`-> `You can also backup ...`

Reply:
:+1:

Reply:
> There are positive ways around this

We don't know that. I made an assumption and it's not validated.